pkg/network/ebpf/c/protocols/http2/decoding-defs.h

```diff
@@ -5,7 +5,19 @@
 
 #include "protocols/http2/defs.h"
 
-#define HTTP2_FRAMES_PER_TAIL_CALL 7
+// Represents the maximum number of frames we'll process in a single tail call in `handle_eos_frames` program.
+#define HTTP2_MAX_FRAMES_FOR_EOS_PARSER_PER_TAIL_CALL 200
+// Represents the maximum number of tail calls to process EOS frames.
+// Currently we have up to 120 frames in a packet, thus 1 tail call is enough.
+#define HTTP2_MAX_TAIL_CALLS_FOR_EOS_PARSER 1
```

Comment:
Not sure I understood the use of HTTP2_MAX_TAIL_CALLS_FOR_EOS_PARSER in the current code change

Reply:
The goal is to make the code ready to be called multiple times by just changing this value (similar behavior exists for the headers parser)

Reply:
I see